docs/DeveloperGuide.adoc

```diff
@@ -297,15 +299,15 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |picky eater |filter food preferences |the food I cook matches my taste
```

Comment:
Phrasing issue:
"the food I cook matches my taste" change to "cook food that matches my taste"

Reply:
thanks!